developers/truffle.md

```diff
@@ -0,0 +1,44 @@
+# Using Truffle Framework with GoChain
+
+You can use [Truffle Framework](https://truffleframework.com/) like you normally do and simply update your `truffle-config.js` configuration file to point to a GoChain network.
+
+Below is an example `truffle-config.js` config file that uses GoChain. To use this, first:
+
+```sh
+npm install truffle-privatekey-provider
+# Set a private key to use that has some GO in its account
+export WEB3_PRIVATE_KEY=0xABCD...
+```
+
+Then you can use this:
+
+```js
+const privateKey = process.env.WEB3_PRIVATE_KEY;
+const PrivateKeyProvider = require("truffle-privatekey-provider");
+const Web3 = require('web3');
+
+module.exports = {
+  networks: {
+    development: {
+      provider: () => new PrivateKeyProvider(privateKey, "https://localhost:8545"),
```

Comment:
Does this work? I would expect `http://`

Reply:
Changed.